tbx/project_styleguide/templates/patterns/atoms/sprites/sprites.html

```diff
@@ -100,4 +100,63 @@
     <symbol id="target" viewBox="0 0 80 80" fill="none">
         <circle cx="40" cy="39.999" r="5.345" fill="var(--color--accent-four)" style="mix-blend-mode:screen"/><circle cx="40" cy="40" r="29.931" stroke="var(--color--accent-two)" stroke-width="2.138" style="mix-blend-mode:screen"/><circle cx="40" cy="39.999" r="18.172" stroke="var(--color--accent-three)" stroke-width="2.138" style="mix-blend-mode:screen"/>
     </symbol>
+
+    {# for animated megaphone icon - megaphone #}
```

Comment:
It's a good thought. You'll probably know more than me on this so if the caching doesn't make it worth it then we can leave it as is. I suppose if more animations are added over time then it might become worth it?

Reply:
I'm honestly not sure so let's leave it for now.